README.md

```diff
@@ -18,6 +18,30 @@ On Linux, macOS and Termux this is available by default.
 
 Windows will require a seperate [installation](https://itsfoss.com/install-bash-on-windows/).
 
+#### Setting up SSH using OpenSSH
```

Comment:
In hindsight I think we should keep explicit setup instructions for SSH excluded. My reasoning being 1. if you've made it this far you probably already know how to do it yourself and 2. If you don't know, you probably won't know to switch out the package manager specification to match your distro. 

I do like that you've added a brief though! Perhaps it is worthwhile mentioning that setting up SSH will differ from device to device, and it's something you'll need to sort out yourself.

Reply:
Sure, good points. Then maybe just point users to the help file's SSH setup link?